features/support/pages/record_of_guidance_preview_page.rb

```diff
@@ -0,0 +1,6 @@
+class RecordOfGuidancePreviewPage < SitePrism::Page
+  set_url_matcher %r{/appointment_summaries/preview}
+
+  element :name, '.name'
```

Comment:
The HTML view of the `OutputDocument` is rendered in the page which includes a paragraph with a class of `name`